credentials/apps/badges/services/processing.py

```diff
@@ -53,6 +53,17 @@ def identify_user(*, event_type, event_payload):
     """
     Identifies event user based on provided keyword arguments and returns the username.
 
+    if (
```

Comment:
This was moved to `credentials/apps/badges/services/revocation.py#process_penalties`.